app/components/containerInspect/containerInspect.html

```diff
@@ -0,0 +1,22 @@
+<rd-header>
+  <rd-header-title title="Container inspect">
+    <i id="loadingViewSpinner" class="fa fa-cog fa-spin" ng-if="state.loaded === false"></i>
+  </rd-header-title>
+  <rd-header-content ng-if="state.loaded">
+    <a ui-sref="containers">Containers</a> &gt; <a ui-sref="container({id: data.Id})">{{ ::data.Name|trimcontainername }}</a> &gt; Inspect
+  </rd-header-content>
+</rd-header>
+
+<div class="row" ng-if="state.loaded">
```

Comment:
No need to check for `state.loaded` here. You should check on `data` (also, see my comment about renaming that field below).